app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/adapters/recycleradapters/RoomsAdapter.kt

```diff
@@ -34,6 +34,7 @@ class RoomsAdapter(private val availableRoomsList: ArrayList<DeviceConnectFragme
                 val roomNameClicked = availableRoomsList[adapterPosition].room
                 var intent = Intent(context, DeviceActivity::class.java)
                 intent.putExtra("roomName", roomNameClicked)
+                intent.putExtra(DeviceActivity().CONNECT_TO, DeviceActivity().TAG_DEVICE_CONNECT_FRAGMENT)
```

Comment:
No need to instantiate activity. Heard of static final variables?

Reply:
Yes, we can do so via using the companion object in kotlin.